src/DssSpell.t.sol

```diff
@@ -1164,24 +1156,13 @@ contract DssSpellTest is DSTest, DSMath {
 
         // Insert new collateral tests here
         checkUNIV2LPIntegration(
-            "UNIV2WBTCDAI-A",
-            GemJoinAbstract(addr.addr("MCD_JOIN_UNIV2WBTCDAI_A")),
-            FlipAbstract(addr.addr("MCD_FLIP_UNIV2WBTCDAI_A")),
-            LPOsmAbstract(addr.addr("PIP_UNIV2WBTCDAI")),
-            OsmAbstract(addr.addr("PIP_WBTC")).src(),
+            "UNIV2DAIUSDT-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_UNIV2DAIUSDT_A")),
+            FlipAbstract(addr.addr("MCD_FLIP_UNIV2DAIUSDT_A")),
+            LPOsmAbstract(addr.addr("PIP_UNIV2DAIUSDT")),
             0x47c3dC029825Da43BE595E21fffD0b66FfcB7F6e, // DAI DSValue
-            true,
+            OsmAbstract(addr.addr("PIP_USDT")).src(),
             false,
-            true
-        );
-        checkUNIV2LPIntegration(
-            "UNIV2AAVEETH-A",
-            GemJoinAbstract(addr.addr("MCD_JOIN_UNIV2AAVEETH_A")),
-            FlipAbstract(addr.addr("MCD_FLIP_UNIV2AAVEETH_A")),
-            LPOsmAbstract(addr.addr("PIP_UNIV2AAVEETH")),
-            OsmAbstract(addr.addr("PIP_AAVE")).src(),
-            OsmAbstract(addr.addr("PIP_ETH")).src(),
-            true,
```

Comment:
Well I made this last spell, and have heard no such directive. It would also require more code to deal with debt ceilings and what not as the ilks are already deployed. I've just opted for the status quo with new ilk integrations as that was easier at the time. Happy to discuss if we want to start testing all ilks all the time.

Reply:
We had only really started tracking pip authorizations. Integration tests have been removed from week-to-week.

I would love to have a suite of tests that we can run across all of deployed MCD, but I suppose it shouldn't go into this repository.

Reply:
or should it...

Reply:
Non full-node plebs trying to test all of this will curse us if we leave them all here.

Reply:
Yeah my thoughts as well. Running integration tests on all collaterals will eat up a lot of time.